pom.xml

```diff
@@ -119,6 +119,9 @@
          -->
         <profile>
             <id>fastest</id>
+            <properties>
+                <cyclonedx.skip>true</cyclonedx.skip>
```

Comment:
Maybe set this property:
https://github.com/eclipse-ee4j/ee4j/blob/bd303c4e8badb547d505db7272f3319a1f4d16a0/parent/pom.xml#L200: https://github.com/eclipse-ee4j/glassfish/blob/bd303c4e8badb547d505db7272f3319a1f4d16a0/parent/pom.xml#L200

This would not add plugin to the build at all and avoid its execution (not only skipping it).

Reply:
The `skipSBOM` property deactivates a profile, therefore it has to be set on maven's command line. If it's set in the `fastest` profile, it's too late to deactivate the sbom. It's not possible to activate or deactive a profile from another profile.

Reply:
Could the sbom profile be deactivated in maven.config?

Reply:
Having `<cyclonedx.skip>true</cyclonedx.skip>` as above in `fastest` is perfect on its own and worth to have it in that profile.